### PR TITLE
remove builder-notify and kafka remnants

### PR DESCRIPTION
**Pull request by pozsgaic**
Signed-off-by: pozsgaic <pozsgai@progress.com>

### CODE REVIEW

**Comment by pozsgaic** [comment]
Original commit that most of the work was done:
commit 3df1f39535d073e2753a37d4399eebc908b22b35
Author: Ian Henry <ianjhenry00@gmail.com>
Date:   Tue Sep 15 11:55:05 2020 -0400

    Adds notification generation to builder api

**Comment by pozsgaic** [comment]
To ensure that the events are left intact, the Builder was tested in dev standalone mode and the following builder-api proxy settings were used to view events:
enable_builder_events = true

File: components/builder-api/habitat/plan.sh [hunk 1]
```diff
@@ -7,7 +7,9 @@ pkg_bin_dirs=(bin)
 pkg_deps=(core/glibc core/openssl core/coreutils core/gcc-libs core/zeromq
 core/libarchive core/curl core/postgresql)
 pkg_build_deps=(core/protobuf-cpp core/protobuf-rust core/coreutils core/cacerts
-core/rust/"$(cat "../../../rust-toolchain")" core/gcc core/git core/pkg-config core/bash core/make core/cmake)
+core/rust/"$(cat "../../../rust-toolchain")" core/gcc core/git core/pkg-config core/cmake)
+#pkg_build_deps=(core/protobuf-cpp core/protobuf-rust core/coreutils core/cacerts
```
Comment (mwrock) [inline components/builder-api/habitat/plan.sh:11]: You probably meant to delete this?

File: components/builder-api/src/server/mod.rs [hunk 2]
```diff
@@ -76,33 +73,27 @@ features! {
...
                        packages: S3Handler::new(config.s3.clone()),
                        github: GitHubClient::new(config.github.clone())?,
                        jobsrv: RpcClient::new(&format!("{}", config.jobsrv)),
                        oauth: OAuth2Client::new(config.oauth.clone())?,
                        memcache: RefCell::new(MemcacheClient::new(&config.memcache.clone())),
                        artifactory: ArtifactoryClient::new(config.artifactory.clone())?,
-                       db,
-                       event_producer: None };
-
-        if feat::is_enabled(feat::EventBus) {
```
Comment (mwrock) [inline components/builder-api/src/server/mod.rs:103]: Since we are doing away with this feature, we should also delete lines 108 and 117 here.

**Review by mwrock — CHANGES_REQUESTED** [review]

File: components/builder-api/habitat/plan.sh (hunk 1 above)
Comment (pozsgaic) [reply components/builder-api/habitat/plan.sh:11]: Yes - will update

File: components/builder-api/src/server/mod.rs (hunk 2 above)
Comment (pozsgaic) [reply components/builder-api/src/server/mod.rs:103]: Agree - will update.

File: components/builder-api/src/server/mod.rs [hunk 3]
```diff
@@ -123,7 +112,6 @@ fn enable_features(config: &Config) {
     }
 
     if feat::is_enabled(feat::List) {
-        println!("Listing possible feature flags: {:?}", features.keys());
```
Comment (mwrock) [inline components/builder-api/src/server/mod.rs:126]: did you intend to delete this?

**Review by mwrock — APPROVED** [review]

**Pull request merged**